internal/utils/hashing_test.go

```diff
@@ -0,0 +1,38 @@
+package utils
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestShouldHashString(t *testing.T) {
+	input := "input"
+
+	sum := HashSHA256FromString(input)
+
+	assert.Equal(t, "c96c6d5be8d08a12e7b5cdc1b207fa6b2430974c86803d8891675e76fd992c20", sum)
```

Comment:
Also hash a different string to check if the hashes differ.

Reply:
Done